core/src/test/java/org/infinispan/statetransfer/RemoteGetDuringStateTransferTest.java

```diff
@@ -54,40 +63,88 @@
    /*
    Summary
 
-   T = initial topology, T1  = rebalance started but we have no data yet, T1.5 rebalance started and we have data, T2  = rebalance finished
-
-   The originator will retry the remote get if node 1 processes the request in T2.
-   If node 1 processes the request in T0 or T1, it still has the entry, and the originator doesn't have to retry.
+   T0   initial topology
+   T1   rehash started
+   T2   rehash finished but state transfer not complete
+   T3   read new, write all topology
+   T4   state transfer completed
 
    | sc     | osc | first request | process request 1 | receive response 1 | retry | process request 2 | receive response 2 |
```

Comment:
Will remove `osc`, but won't add another columns; the prefixes show clearly which node does that.